calico-enterprise/release-notes/index.mdx

```diff
@@ -50,6 +50,10 @@ For more information, see [Helm installation reference](../reference/installatio
 
 <!-- BELOW ARE 3.18 EP2 features -->
 
+### Support for Kubernetes v1.28
```

Comment:
This needs to be updated by Phil.

Reply:
I've added this to PR for 3.18-2.

Reply:
Yes, RN will be handled in the 3.18-2 release PR.